### PR TITLE
Portfolio Milestone 8

### PR DESCRIPTION
**Pull request by ambrose-kibet**
### in this milestone we : 
- add a modal to index.html
- add modal functionality to index.js

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -159,12 +172,12 @@ <h2 class="card-title">Multi-Post Stories</h2>
...
               </div>
               <div class="card-footer">
-                <button class="btn">See Project</button>
+                <button class="btn pop-toggler">See Project</button>
               </div>
             </div>
           </article>
           <div class="project-cards-container">
-            <article class="project-card-variant printing">
+            <article class="project-card-variant printing" data-name="printing">
```
Comment (ha-manel) [inline index.html]: - [x] Kindly remove the projects cards from the HTML and add them dynamically using JavaScript as mentioned in the project requirements 👇 

image

File: index.js [hunk 2]
```diff
@@ -14,3 +68,47 @@ navLinks.forEach((link) => {
...
+                </h3>
+                <ul class="tech-container">
+                ${tech.map((tech) => `<li class="tech">${tech}</li>`)}
+                </ul>
+                <div class="project-card">
+                  <div class="pop-img-container">
+                    <img
+                      src="${src}"
+                      alt="${title}"
+                      class="card-header-img"
```
Comment (ha-manel) [inline index.js]: - [x] Good job with the design of the modal window 👏 Kindly add a dark overlay behind the modal with a slight blur (for both desktop and mobile view) as shown in the Figma template 👍 

## Your design:
image

## Figma design:
image

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi Team👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Modal window is working properly ✔️ 

## Required Changes:

- [x] Kindly add the MIT file to your project because the link in your README file is broken 👍 
- [x] Please consider following best practices when you write commit messages, they should be capitalized sentences in the imperative form, example: "Add modal window" instead of "modal window added" ... etc. 💯 

_Kindly check the comments below_


------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by cynthiainga — APPROVED** [review]
## STATUS APPROVED ✔️ ✔️ 

Hi @ambrose-kibet, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Correct Github flow ✔️ 

## Optional suggestions

- [OPTIONAL] As mentioned by the previous reviewer kindly add the `MIT` file to your project because the link that you provided in your README file is broken 🙃 
- [OPTIONAL] Also some of your commit messages are in the past tense and not capitalized, the commit message should be written in the imperative tense and capitalized. [Read more about how to write descriptive commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8). Kindly apply this rule in your next commit messages 🙃 😉

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**